modules/common-workflows/pages/workflow-inplace-sles-upgrade.adoc

```diff
@@ -0,0 +1,204 @@
+[[workflow-inplace-sles-upgrade]]
+
+= In-place Upgrade of {sles} with {productname}
```

Comment:
```suggestion
= In-place upgrade of {sles} with {productname}
```
Maybe, with new sections or chapters we can try to be consistent with capitalizing titles (= using sentence style)